src/components/material/infomedia/InfomediaModalBody.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { GetInfomediaQuery } from "../../../core/dbc-gateway/generated/graphql";
+
+export interface InfomediaModalBodyProps {
+  infomediaArticle: GetInfomediaQuery["infomedia"]["article"];
+}
+
+const InfomediaModalBody: React.FunctionComponent<InfomediaModalBodyProps> = ({
+  infomediaArticle
+}) => {
+  return (
+    <article>
+      {infomediaArticle?.headLine && <h2>{infomediaArticle?.headLine}</h2>}
```

Comment:
```suggestion
      {infomediaArticle?.headLine && <h2>{infomediaArticle.headLine}</h2>}
```

The second null chaining should not be necessary with the first check in mind.